HW_7/view.py

```diff
@@ -0,0 +1,14 @@
+def show_menu():
+    question = 'Введите необходимую команду:\
+        \n1 - Показать все контакты\
+        \n2 - Открыть файл с контактами\
+        \n3 - Записать файл с контактами\
+        \n4 - Добавить контакт\
+        \n5 - Поиск по контактам\
+        \n0 - Выход\
+        \n'
```

Comment:
Смотрите, Анастасия, при каждом вызове этой функции у вас будет определяться эта переменная, но зачем? Просто поместите ее в глобальную область видимости и все.
```suggestion
question = 'Введите необходимую команду:\
    \n1 - Показать все контакты\
    \n2 - Открыть файл с контактами\
    \n3 - Записать файл с контактами\
    \n4 - Добавить контакт\
    \n5 - Поиск по контактам\
    \n0 - Выход\
    \n'


def show_menu():
    command = input(question)
    return command
```